coalib/output/ConsoleInteraction.py

```diff
@@ -322,6 +323,8 @@ def print_results_formatted(log_printer,
 
             for range in result.affected_code:
                 format_args['affected_code'] = range
+                format_args['source_lines'] = range.affected_source(args[0])
```

Comment:
args[0] isn't well readable, just add the file_dict arg above and use it directly